src/services/ui/src/pages/actions/WithdrawPackage.tsx

```diff
@@ -33,42 +51,171 @@ const addlInfoInstructions: Record<SetupOptions, ReactElement> = {
+          </p>
+        </ActionFormIntro>
+        {/* Package ID and type info */}
+        <PackageInfo item={item} />
+        {/* Attachments */}
+        <h3 className="font-bold text-2xl font-sans">Attachments</h3>
+        {
+          attachmentInstructions[
+            item!._source.planType as string as SetupOptions
+          ]
+        }
+        <AttachmentsSizeTypesDesc faqLink={"/faq"} />
+        {attachments.map(({ name, label, required }) => (
+          <FormField
+            key={String(name)}
+            control={form.control}
+            name={`attachments.${String(name)}` as Path<typeof schema>}
+            render={SlotAttachments({
+              label: (
+                <>
+                  {label}
+                  {required ? <RequiredIndicator /> : ""}
+                </>
+              ),
+              message: <FormMessage />,
+              className: "my-4",
+            })}
+          />
+        ))}
+        {/* Additional Info */}
+        <FormField
+          control={form.control}
+          name={"additionalInformation" as Path<typeof schema>}
+          render={SlotAdditionalInfo({
+            label:
+              addlInfoInstructions[
+                item!._source.planType as string as SetupOptions
+              ],
+            description: "4,000 characters allowed",
+            className: "pt-6",
+            required: false,
+          })}
+        />
+        {/* Error banner */}
+        {Object.keys(form.formState.errors).length !== 0 && (
+          <Alert className="my-6" variant="destructive">
+            Input validation error(s)
+            <ul className="list-disc">
+              {Object.values(form.formState.errors).map(
+                (err, idx) =>
+                  err?.message && (
+                    <li className="ml-8 my-2" key={idx}>
+                      {err.message as string}
+                    </li>
+                  )
+              )}
+            </ul>
+          </Alert>
+        )}
+        {/* Pre-submit message banner */}
+        <Alert variant={"infoBlock"} className="my-2 w-full flex-row text-sm">
+          <Info />
+          <p className="ml-2">
+            Once complete, you will not be able to resubmit this package. CMS
+            will be notified and will use this content to review your request.
+            If CMS needs any additional information, they will follow up by
+            email.
+          </p>
+        </Alert>
+        {/* Buttons */}
+        <div className="flex gap-2 my-8">
+          <Button
+            type={"button"}
+            onClick={() => {
+              setModalContent({
+                header: "Withdraw package?",
+                body: `The package ${item._source.id} will be withdrawn.`,
+                acceptButtonText: "Yes, withdraw package",
+                cancelButtonText: "Cancel",
+              });
+              setModalOnAccept(() => confirmOnAccept);
```

Comment:
I haven't tested yet, but this looks like it needs to be updated to:

```js
setModalOnAccept(confirmOnAccept);
```

Reply:
This works as-intended. To properly test/confirm, change the `() => confirmOnAccept` to `confirmOnAccept` and try the Cancel button on this, or the WithdrawRai, forms.